File: src/main/scala/ide/view/ui/frame/MonoGFrame.scala
package az.rock
package ide.view.ui.frame

import java.awt.{BorderLayout, Dimension}

abstract class MonoGFrame(name:String) extends GFrame(name:String){

  {
    this.setSize(new Dimension(700,600));
    this.setLayout(new BorderLayout());
    this.setLocationRelativeTo(null)
  }
}

File: src/main/scala/ide/view/ui/frame/MonoGFrame.java
package az.rock.ide.view.ui.frame;

import java.awt.*;
import java.awt.geom.RoundRectangle2D;

public abstract class MonoGFrame extends GPrimaryProcessFrame {

    {
        super.setSize(new Dimension(900,700));
        super.setLayout(new BorderLayout());
        super.setLocationRelativeTo(null);
    }

    public MonoGFrame(String name){
        super(name);
    }

}